orchestra/todos/decorators.py

```diff
@@ -0,0 +1,33 @@
+import logging
```

Comment:
1) This feels more generic than just `todos`
2) Do we need this? We've already got a pattern for decorating API requests.

Reply:
@marcua Maybe I am using a really old api as my reference - https://github.com/b12io/orchestra/tree/master/orchestra/project_api 

Can you point me to the pattern for decorating API requests?

Reply:
Given that https://github.com/b12io/orchestra/tree/master/orchestra/project_api/decorators.py already exists, you shouldn't duplicate it. Either import from it, or move it to a more central place and import in both locations.

If there are any other copy/pasted modules in this PR, do the same for them.